Makefile

```diff
@@ -182,7 +182,8 @@ BUF := $(LOCALBIN)/buf-$(BUF_VER)
 $(BUF): | $(LOCALBIN)
 	$(call go-install-tool,$(BUF),github.com/bufbuild/buf/cmd/buf,$(BUF_VER))
 
-GO_API_VER := v1.33.0
+
+GO_API_VER := v1.46.0
```

Comment:
ie our Temporal API version.

Reply:
I was thinking the same thing but it wasn't immediately obvious to me how to pull it off. Let me try that command! I think it'd be good to have them in-sync.

Reply:
Pushed! Only downside I can see is that it's evaluated for every Makefile command. I couldn't figure out how to make that lazy. But it's very fast ...

Reply:
I'll merge it now, but feel free to suggest an improvement and I'll follow-up.